src/scenes/home/scholarship/preview/preview.css

```diff
@@ -0,0 +1,11 @@
+.cmon_man {
+  /*border-bottom: gray solid 1px;
+  margin: 20px 15px 0px 15px;*/
+  position: relative;
```

Comment:
Why is this CSS class named `.cmon_man` ? This should be clarified.

Commented out styles can be removed as well.

Reply:
Thanks, I'll rename and remove the commented out styles